libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.Foreach.schema

```diff
@@ -1,14 +1,14 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "Foreach Action",
-    "description": "Action which executes actions per item in a collection.",
+    "title": "For-each loop",
```

Comment:
>loop [](start = 23, length = 4)

For-each item?

Reply:
Both the fact that it this action depend on a condition and that it loops are the important things to convey here. See also principles below.